### ISSUE TITLE
Token

### ISSUE THREAD

**Issue by NoemiNL** [cxii-dev/script.tvshowtime#5]
Hi,

When I installed and tried to get  token I got the following message at the page

"Whoops
Our put.io integration is no longer available."

Is there a way around this to make the add-on still work?

Thanks in advance,
Noëmi

**Comment by cxii-dev** [comment]
Hi,

Sorry for that. I received an email this morning from TVST. They have changed the way to authenticate. So I will change the code as soon as possible.

**Comment by NoemiNL** [comment]
Thanks for the information, looking forward to the update.

**Comment by raspdealer** [comment]
The version 10 fixes this ?

**Comment by cxii-dev** [comment]
the 11 fixes
I will test this build before submitting to kodi repo

**Comment by cxii-dev** [comment]
the 1.0.12 is submitted to kodi repo

**Comment by NoemiNL** [comment]
Maybe you can also place the precise URL in the documentation of the plugin. ;-)
After install the plugin shows: BIT.LY/TVSHOWTIME
Instead this should be http://bit.ly/TVShowTime  

The all Caps made a difference.

When you commented that the token issue was fixed I checkt but still got the same message. Only after I read in another issue the correct URL it became clear that Capitals where important. 

Anyhow, thanks for fixing! :-)

**Comment by cxii-dev** [comment]
Ok, thank you
